app/PharBuilder.php

```diff
@@ -455,7 +455,7 @@ protected function addDir($directory, $excludes = array())
              *
              * @var \Symfony\Component\Finder\SplFileInfo $file
              */
-            $this->addFile($directory . DIRECTORY_SEPARATOR . $file->getRelativePathname());
+            $this->addFile(rtrim($directory, '\/') . DIRECTORY_SEPARATOR . $file->getRelativePathname());
```

Comment:
Why does it end with a slash on Windows? Shouldn't this be normalized in the finder?